components/LoadingFull.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import styles from './styles';
```

Comment:
it's not used right now (maybe we should install `eslint` in the project with `no-unused-imports` rule 🤔 but let's focus on implementing UI for now), so I guess you could remove it.

However, I don't know why you're passing `styleProp` to this component. It would be much better to  hardcode it in this file (as it should be always the same).

BTW -> if you want to add type for `style` then you can use `StyleProp<ViewStyle>`, both need to be imported from `react-native`